### ISSUE TITLE
cannot cycle through history items

### ISSUE THREAD

**Issue by rickysarraf** [Keruspe/GPaste#169]
In the GPaste gui, there does not seem to be a simple way (in fact, as I see, no way) to browse through the history items.

I've set my GPaste history to be 200. But all I see is first 20 items.  How do I browse through the entire history ? Please see screenshot.

![image](https://cloud.githubusercontent.com/assets/65480/18435691/65bcddbe-7912-11e6-992c-d5cbe3782c93.png)

**Comment by Keruspe** [comment]
For now, you can't.
In version 3.21.9x, there's some pagination in the gnome-shell extension, which I'll add to the gui too before doing the 3.22 final release.

**Comment by Keruspe** [comment]
You can access further items using the search, though, fwiw.

**Comment by rickysarraf** [comment]
Yes. That's how I've been working around it. But the GUI lacked the flow, hence this bug report.

**Comment by Keruspe** [comment]
I really had less time than I wish I could and 3.22 is already really late for release, so I'm delaying this for 3.24, sorry.
